src/components/UI/index.js

```diff
@@ -17,3 +17,4 @@ export { default as Oppertunities } from './oppertunities';
 export { default as IssuerSteps } from './issuer-steps';
```

Comment:
No need to add brackets and `default as` and the name could be anything you want.